src/components/Navigation.tsx

```diff
@@ -0,0 +1,36 @@
+import cn from 'classnames';
+import { NavLink } from 'react-router-dom';
+
+export const Navigation = () => {
+  const getLinkClass = ({ isActive }: { isActive: boolean }) => cn(
+    'navbar-item',
+    { 'has-background-grey-lighter': isActive },
+  );
```

Comment:
It is better to define this function outside the component so that it is not created every time a new render is performed.